build.sbt

```diff
@@ -15,10 +15,10 @@ val http4sJdkHttpClientVersion = "0.9.0"
 val jwtVersion                 = "5.0.0"
 val logbackVersion             = "1.4.7"
 val log4catsVersion            = "2.6.0"
-val lucumaCoreVersion          = "0.76.0"
+val lucumaCoreVersion          = "0.77.0"
 val lucumaGraphQLRoutesVersion = "0.6.4"
-val lucumaItcVersion           = "0.12.1"
-val lucumaSsoVersion           = "0.5.10"
+val lucumaItcVersion           = "0.13.1"
```

Comment:
Do you need 0.14 for core 0.77.1?

Reply:
I think 0.13.1 gets me to core 0.77.0, which is what I needed.